ngraph/frontend/onnx_import/src/op/upsample.cpp

```diff
@@ -29,47 +29,53 @@ namespace ngraph
-                                        const int64_t opset)
+                                        const unsigned op_version)
                 {
-                    const std::unordered_set<std::string> modes_opset1 = {"nearest", "bilinear"};
-                    const std::unordered_set<std::string> modes_opset7 = {"nearest", "linear"};
-                    const auto& supported_modes = opset < 7 ? modes_opset1 : modes_opset7;
-                    const bool is_mode_supported =
-                        (std::find(supported_modes.begin(), supported_modes.end(), mode) !=
-                         supported_modes.end());
-
-                    if (!is_mode_supported)
+                    const std::unordered_set<std::string> modes_v1 = {"nearest", "bilinear"};
+                    const std::unordered_set<std::string> modes_v7 = {"nearest", "linear"};
+                    const auto& supported_modes = op_version < version_7 ? modes_v1 : modes_v7;
+
+                    if (std::find(supported_modes.cbegin(), supported_modes.cend(), mode) ==
+                        supported_modes.cend())
                     {
                         std::string supported_modes_str = "";
                         for (const auto& mode_name : supported_modes)
                         {
                             supported_modes_str += (mode_name + ", ");
                         }
                         CHECK_VALID_NODE(node,
-                                         is_mode_supported,
+                                         false,
                                          mode,
                                          " - this type of interpolation mode is not supported."
                                          " Choose one of the following modes: ",
                                          supported_modes_str);
                     }
-                    return is_mode_supported;
                 }
 
-                using Transform_mode = ngraph::op::v4::Interpolate::CoordinateTransformMode;
-                using Nearest_mode = ngraph::op::v4::Interpolate::NearestMode;
-                using InterpolateMode = ngraph::op::v4::Interpolate::InterpolateMode;
-                using ShapeCalcMode = ngraph::op::v4::Interpolate::ShapeCalcMode;
-
-                InterpolateMode convert_mode(const std::string& mode_str)
+                default_opset::Interpolate::InterpolateAttrs get_attributes(std::string mode)
                 {
-                    InterpolateMode result = InterpolateMode::nearest;
-                    if (mode_str == "linear" || mode_str == "bilinear")
-                    {
-                        result = InterpolateMode::linear_onnx;
-                    }
-                    return result;
+                    using InterpolateMode = default_opset::Interpolate::InterpolateMode;
+                    using Transform_mode = default_opset::Interpolate::CoordinateTransformMode;
```

Comment:
For me `using` like `using Transform_mode` makes harder to recognize origin of these modes and
`default_opset::Interpolate::CoordinateTransformMode` is maybe longer but more meaningful during code reading.
Just my opinion, no changes required.